server/views/pages/contacts/manage/prisonerSearchResults.njk

```diff
@@ -84,7 +84,7 @@
             {% set rows = [] %}
             {% for result in results.content %}
                 {% set prisonerNameHtml %}
-                   <a href="../list/{{journey.id}}?prisoner={{ result.prisonerNumber }}" class="govuk-link govuk-link--no-visited-state bapv-result-row">
+                   <a href="/prisoner/{{ result.prisonerNumber }}/contacts/list/{{journey.id}}" class="govuk-link govuk-link--no-visited-state bapv-result-row">
                       {{ (result.lastName + ", " + result.firstName) | convertToTitleCase }}
```

Comment:
Using the prisoner search result here, but (like in other places) we could avoid recalling on every page and use the stored journey values? Fine for now, as mini profile also uses result at present.

Reply:
This is the list of results so think it's still not in a prisoner context here but I get your meaning.